gui/src/3D/controls/holographicSlate.ts

```diff
@@ -42,12 +42,12 @@ export class HolographicSlate extends ContentDisplay3D {
     /**
      * Dimensions of the slate
      */
-    public dimensions = new Vector3(0.7, 0.4, 0.001);
+    public dimensions = new Vector3(21.875, 12.5, 0.001);

```

Comment:
just wondering - is that a breaking change, or does it get adjusted correctly?

Reply:
It gets adjusted correctly when the GUI3DManager flag is set. The slate will appear larger by default though, but this shouldn't be an issue since it was pre-scaled to an XR-appropriate size when it was first added, and should be larger without the flag to match the size of the other controls. Same goes for the NearMenu/HandMenu controls.

Reply:
I don't really consider it a breaking change since this class was introduced in 5.0

Reply:
All that being said, the default size is now much larger for both the HolographicSlate and Near/Hand Menu (since the internal scale-down on the menus has been removed).
 I don't expect developers to use these elements unless designing for an XR experience though, in which case they would want to use the new GUI3DManager flag, so I'm not too concerned with the defaults being very large.